roxctl/deployment/check/check.go

```diff
@@ -187,9 +198,24 @@ func (d *deploymentCheckCommand) Check() error {
 }
 
 func (d *deploymentCheckCommand) checkDeployment() error {
-	deploymentFileContents, err := os.ReadFile(d.file)
-	if err != nil {
-		return errors.Wrapf(err, "could not read deployment file: %q", d.file)
+	var deploymentFileContents []byte
+	if d.file != "" {
+		var err error
+		deploymentFileContents, err = os.ReadFile(d.file)
+		if err != nil {
+			return errors.Wrapf(err, "could not read deployment file: %q", d.file)
+		}
+	} else {
+		for _, file := range d.files {
+			fileContents, err := os.ReadFile(file)
+			if err != nil {
+				return errors.Wrapf(err, "could not read deployment file: %q", file)
+			}
+			if len(deploymentFileContents) > 0 {
+				deploymentFileContents = append(deploymentFileContents, "---\n"...)
```

Comment:
You probably want to add a line break in the beginning as well in case the input has no endline. 
```suggestion
				deploymentFileContents = append(deploymentFileContents, "\n---\n"...)
```

Reply:
Is a `.yaml` still formatted correctly if there's an empty line before the `---` separating two deployments? I might need to test this, and if that's the case perhaps check whether input files have a newline at EOF.

Reply:
I think it should be fine. Adding multiple lines between separators [here](https://www.yamllint.com/) seems to result in a valid yaml.